Dockerfile

```diff
@@ -48,7 +48,7 @@ RUN apt-get update \
   && rm -rf /var/lib/apt/lists/* \
   && sed -i -E 's/^(USERGROUPS_ENAB\s+)yes$/\1no/' /etc/login.defs \
   && localedef -i en_US -f UTF-8 en_US.UTF-8 \
-  && useradd --create-home --shell /bin/bash --user-group linuxbrew \
+  && useradd -u 1001 --create-home --shell /bin/bash --user-group linuxbrew \
```

Comment:
Why 1001?

Reply:
This is because `useradd` will yield to different user ids in different ubunti versions and we need a stable number so we can use cache volumes below here: https://github.com/Homebrew/brew/pull/17584/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R69

Reply:
Can you add a comment explaining this? Also, ideally this would be a variable rather than hardcoding `1001` in a few places.

Reply:
done